src/appleseed-max-impl/appleseedoslplugin/oslshaderregistry.cpp

```diff
@@ -439,12 +442,12 @@ void OSLShaderRegistry::create_class_descriptors()
         for (auto& param_info : shader.m_params)
         {
             param_info.m_max_param.m_max_ctrl_id = ctrl_id++;
-            param_info.m_max_param.m_max_param_id = param_id;
-
-            shader.m_string_map.insert(std::make_pair(string_id, utf8_to_wide(param_info.m_max_param.m_max_label_str)));
 
             if (param_info.m_max_param.m_has_constant)
```

Comment:
Unrelated to this PR but I wonder: what does "has constant" mean? Should it be `m_is_constant`? (In which case it should also be `m_is_connectable`, etc.)

Reply:
Constant is color or float parameter, which can or cannot be textured. I can rename it to m_is_constant

Reply:
Now I think `m_is_constant` is misleading because parameter can be a constant and be connectable.. Is it ok you think to have both `m_is_constant` and `m_is_connectable` set to true?

Reply:
I was just wondering about the choice of verb ("has" vs "is"). I don't know the code enough to know which verb is more accurate.